definitions/divorce/json/CaseEvent.json

```diff
@@ -207,6 +207,7 @@
     "DisplayOrder": 4,
     "PreConditionState(s)": "AosStarted",
     "PostConditionState": "AosOverdue",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/petition-updated",
     "RetriesTimeoutURLSubmittedEvent": "120,120",
```

Comment:
also this retry, `RetriesTimeoutURLSubmittedEvent`

Reply:
is it not `RetriesTimeoutURLSubmittedEvent` already? @qzhou-hmcts

Reply:
sorry, this is right, I don't know why I saw `AboutToSubmit` here